### ISSUE TITLE
Issue about the t_loss_mpl

### ISSUE THREAD

**Issue by Xiao0728** [kekmodel/MPL-pytorch#26]
Hi, 

I was trying to understand the loss procedure of the mpl loss: 
[_, hard_pseudo_label = torch.max(t_logits_us.detach(), dim=-1)](https://github.com/kekmodel/MPL-pytorch/blob/main/main.py#L217)
[t_loss_mpl = dot_product * F.cross_entropy(t_logits_us, hard_pseudo_label)](https://github.com/kekmodel/MPL-pytorch/blob/main/main.py#L218)

Here, why do we compare t_logits_us with itself using cross-entropy? Thanks.

**Comment by mxqmxqmxq** [comment]
why？
